OctoKit/Issue.swift

```diff
@@ -58,25 +104,109 @@ public extension Octokit {
             }
         }
     }
+    
+    /**
+     Fetches an issue in a repository
+     - parameter owner: The user or organization that owns the repository.
+     - parameter repository: The name of the repository.
+     - parameter number: The number of the issue.
+     - parameter completion: Callback for the outcome of the fetch.
+     */
+    public func issue(owner: String, repository: String, number: Int, completion: (response: Response<Issue>) -> Void) {
```

Comment:
I forgot something. If the Issue of would be treated as a String then the RequestKit PR is no longer needed. Pagination is also done using strings and I would prefer that.